src/app/pages/account/groups/group-list/group-list.component.ts

```diff
@@ -137,36 +139,58 @@ export class GroupListComponent implements OnDestroy {
         label : helptext.group_list_actions_label_delete,
         onClick : (members_delete) => {
           const self = this;
-          const conf: DialogFormConfiguration = {
-            title: helptext.deleteDialog.title,
-            message: helptext.deleteDialog.message + `<i>${members_delete.group}</i>?`,
-            fieldConfig: [],
-            confirmCheckbox: true,
-            saveButtonText: helptext.deleteDialog.saveButtonText,
-            preInit: function () {
-              if (self.ableToDeleteAllMembers(members_delete)) {
-                conf.fieldConfig.push({
-                  type: 'checkbox',
-                  name: 'delete_users',
-                  placeholder: T(`Delete all ${members_delete.users.length} users with this primary group?`),
-                  value: false,
-                });
+          this.loader.open();
+          self.ws.call("user.query", [[["group.id", "=", members_delete.id]]]).subscribe(
+            (usersInGroup: User[]) => {
+              this.loader.close();
+
+              const conf: DialogFormConfiguration = {
+                title: helptext.deleteDialog.title,
+                message: helptext.deleteDialog.message + `<i>${members_delete.group}</i>?`,
+                fieldConfig: [],
+                confirmCheckbox: true,
+                saveButtonText: helptext.deleteDialog.saveButtonText,
+                preInit: function () {
+                  if (usersInGroup.length > 0) {
```

Comment:
Let's change it to [return early](https://softwareengineering.stackexchange.com/questions/18454/should-i-return-from-a-function-early-or-use-an-if-statement), i.e.

```typescript
if (!usersInGroup.length) {
  return
}

...
everything else
```